### DISCUSSION TITLE
是否还需要保留豆瓣相关

### DISCUSSION THREAD

**Discussion by ted423** [ronggang/PT-Plugin-Plus#984] [General]
https://github.com/ronggang/PT-Plugin-Plus/issues/870
https://github.com/ronggang/PT-Plugin-Plus/issues/777
目前感觉保证豆瓣的可用性没有太大意义

**Comment by MewX** [comment]
豆瓣重度用户表示从来没用过这个功能😂

**Comment by ted423** [comment]
https://github.com/ronggang/PT-Plugin-Plus/issues/982
